demo/0.5.0/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
```

Comment:
I don't think we to need to continue to push the packaging of the new release here. Now we have it on npm :smile:
Just need to update the old demo to use it 😉

Reply:
Currently, we have to push the demo and update resources used by some examples.
See https://github.com/process-analytics/bpmn-visualization-js/issues/599 to decide if/when we update these examples to use a published package.